guides/slack.md

```diff
@@ -0,0 +1,67 @@
+# Slack
+
+## Purpose of this guide
+
+As of writing, we've grown to over 150 team members and we're also operating almost fully remotely. As such, Slack is one of our most important cross company tools and used extensively by everyone.
+This guide is not intended to be a set of rules around using slack, this document is a guide to how we use slack. This should help everyone understand how to get the most out of slack while also being able to minimise distractions, notification noise and control their own focus time, without missing important updates from their team and the wider organisation.
+
+# Profiles
+
+Please complete your profile fully on joining slack - this means a suitable profile picture (it does not have to be a picture of you, but a clear picture of your face is ideal), your full name and a representative display name.
+
+You can optionally fill in the pronunciation guide for your name and add your preferred pronouns if you feel comfortable doing so.
+
+Please also include your up to date email address and group/role, and the location you are based.
```

Comment:
An example of a shorter way of writing this section, [as discussed](https://github.com/madetech/handbook/pull/424#issuecomment-754557513).

> [Filling in your profile](https://slack.com/intl/en-gb/help/articles/204092246-Edit-your-profile) helps other people find you on Slack.
> 
> Your profile includes:
> - Your full name
> - Your display name (which could be a nickname or a short name)
> - A picture of your face
> - How to say your name
> - Which pronouns you prefer
> 
> It's OK to not fill in all this information if you don't feel comfortable sharing it.

Reply:
Which of those bullet points is it ok not to fill in?

Reply:
I was happy for this to be any/all of them, if it makes the person uncomfortable. My first thought is "surely everyone would be comfortable sharing X", but my second though is "I'm not everyone". 

I think it's clear for anyone not sharing their name/face that it'll make it harder for people to find you? I'm happy with people making that decision for themselves, and getting feedback on it so they can tweak how to present themselves, if needed.

I almost amended the last sentence to be ", but it'll make people harder to find you.", but it felt like weird pressure. Not sure what's best here.

Reply:
I'm personally ok with setting name and _a picture_ along with email and location, at a minimum. This allows each slack user to be distinct and recognisable, while providing the minimum necessary information for colleagues to find and interact with you. It is _ok_ for us to outline a necessary minimum, that's not a bad thing.
The rest of the details as optional feels reasonable to me. I reworded to reflect that more clearly, but did not move to bullet points. Let me know what you think.

Reply:
My edit was more about reducing the word count than how strongly we recommend picture/name/anything. I could suggest an edit in that structure that does more strongly recommend name/picture/email/location, but I don't know if how I've structured it appeals.

It'd be good to add the [link to the slack guidance for where to find the edit button](https://slack.com/intl/en-gb/help/articles/204092246-Edit-your-profile) if you don't want to change this section significantly.

Reply:
Moved to bullets but also added a bit of clarity on why (I) feel certain profile fields should be required, let me know what you think @kjdchapman

Reply:
I like it! I have similar ideas for other sections, but I'm conscious it's better to merge + tweak as we go (and I've run out of time today to write more things :sweat_smile:)